gecon/GECON-01.DATA

```diff
@@ -0,0 +1,301 @@
+REGIONS
+------------------------------------------------------------------------------------------------
+
+
+------------------------------------------------------------------------------------------------
+SOLUTION
+------------------------------------------------------------------------------------------------
+
+
+RPTRST
+  'BASIC=2' 'PBPD' /
+
+EQUIL
+-- Datum    P     woc     Pc    goc    Pc  Rsvd  Rvvd
+ 2023.00  340.0  2100.0   0.0  2100.0  0.0   1     1   0 /  
+
+
+PDVD
+ 2000.0 125.0  
+ 2100.0 125.0  /
+
+PBVD
+ 2000.0 125.0  
+ 2100.0 125.0  /
+
+
+------------------------------------------------------------------------------------------------
+SUMMARY
+------------------------------------------------------------------------------------------------
+
+
+WBHP
+ /
+ 
+WGPR
+ /
+ 
+FPR
+
+FGPR
+
+GGPR
+/
+
+WGPT
+/
+
+WOPT
+/
+
+
+
+------------------------------------------------------------------------------------------------
+SCHEDULE
+------------------------------------------------------------------------------------------------
+
+TUNING
+ 1.0 5.0 /
+ /
+ /
+ 
+RPTRST
+ 'BASIC=2' /
+  
+
+GRUPTREE
+  'PLAT-A'  'FIELD'   /
+  'M1'      'PLAT-A'  /
+  'M2'      'PLAT-A'  /
+  'NE'      'PLAT-A'  /
+/  
+  
+
+GCONPROD
+ 'PLAT-A'   'GRAT'   2*   1250000   1*   'RATE'  /  
+/
+
+GECON
+ 'M1'  1*  200000.0  /
+ 'M2'  1*  200000.0  /
+ 'NE'  1*  200000.0  /
+/ 
+ 
+WELSPECS
+ 'P1'  'NE'   2    8   1*      'GAS'  0.00      'STD'     'SHUT'      'YES'    0      'SEG' /
+ 'P2'  'NE'   3   10   1*      'GAS'  0.00      'STD'     'SHUT'      'YES'    0      'SEG' /
+/
+
+COMPDAT
+ 'P1'   2  8  1 3  'OPEN'   0    1*  0.216  3*   'Z' 1* /
+ 'P2'   3 10  1 3  'OPEN'   0    1*  0.216  3*   'Z' 1* /
+/
+
+WCONPROD
+ 'P1'  'OPEN'   'GRAT'  2*   500000.0  2*   85.0 /
+ 'P2'  'OPEN'   'GRAT'  2*   500000.0  2*   85.0 /
+/
+
+DATES
+ 1 'DEC' 2018 /
+/
+
+WELSPECS
+ 'P3'  'M1'   2    3   1*      'GAS'  0.00      'STD'     'SHUT'      'YES'    0      'SEG' /
+/
+
+COMPDAT
+ 'P3'   2  3 1 5    'OPEN'   0    1*  0.216  3*   'Z' 1* /
+/
+
+WCONPROD
+ 'P3'  'OPEN'   'GRAT'  2*   500000.0  2*   85.0 /
+/
+
+DATES
+ 1 'JAN' 2019 /
+/
+
+WELSPECS
+ 'P4'  'M1'   7   4   1*      'GAS'  0.00      'STD'     'SHUT'      'YES'    0      'SEG' /
+/
+
+COMPDAT
+ 'P4'   7  4   1 5    'OPEN'   0    1*  0.216  3*   'Z' 1* /
+/
+
+WCONPROD
+ 'P4'  'OPEN'   'GRAT'  2*   500000.0  2*   85.0 /
+/
+
+DATES
+ 1 'FEB' 2019 /
+/
+
+WELSPECS
+ 'P5'  'M2'   9   8   1*      'GAS'  0.00      'STD'     'SHUT'      'YES'    0      'SEG' /
+/
+
+COMPDAT
+ 'P5'   9  8  1  5   'OPEN'   0    1*  0.216  3*   'Z' 1* /
+/
+
+WCONPROD
+ 'P5'  'OPEN'   'GRAT'  2*   500000.0  2*   85.0 /
+/
+
+DATES
+ 1 'MAR' 2019 /
+ 1 'APR' 2019 /
+ 1 'MAY' 2019 /
+ 1 'JUN' 2019 /
+ 1 'JUL' 2019 /
+ 1 'AUG' 2019 /
+ 1 'SEP' 2019 /
+ 1 'OCT' 2019 /
+ 1 'NOV' 2019 /
+ 1 'JAN' 2020 /
+ 1 'FEB' 2020 /
+ 1 'MAR' 2020 /
+/  
+ 
+END
+
+ 
+--INCLUDE
+-- 'include/hist_schedule.sch' /
```

Comment:
Will we ever switch to using this schedule?  If not, maybe we can remove this `INCLUDE` statement instead of just disabling it?

Reply:
yes we can.